elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java

```diff
@@ -95,10 +95,25 @@ private void constructJobBootstraps(final ElasticJobProperties elasticJobPropert
         }
     }
     
+    private void jobExtraConfigurations(final JobConfiguration jobConfig, final TracingConfiguration<?> tracingConfig) {
+        if (null == tracingConfig) {
+            return;
+        }
+        TracingProperties tracingProperties = applicationContext.getBean(TracingProperties.class);
```

Comment:
What if user configured both `include` and `exclude`? I prefer throwing an exception.

Reply:
you are right